src/presenter/presenter.py

```diff
@@ -0,0 +1,187 @@
+from PyQt5 import uic
+from src import UI_DIR, CONFIG_DIR
+import json
+from PyQt5 import QtWidgets, QtCore
+import os
+import nibabel as nib
+from src import DATA_DIR, OUT_DIR
+from src.presenter.utils import view_manager, store_image, get_image, enable_threading
+from src.view import ui
+DATA_DIR
```

Comment:
precommit didnt wanted to allow import of DATA_DIR as DATA_DIR was not used in functions below (only one time but with 'global' specified. I worked around this little issue by adding this line.

Reply:
But if you do not use it in your code, why did you import it?

Reply:
No I use it here:
`def browse_image(widget):
    global DATA_DIR
    dialog = QtWidgets.QFileDialog()
    filename, _ = dialog.getOpenFileName(widget.node.graph, "Select a file...", DATA_DIR)
    DATA_DIR = os.path.dirname(filename)`

But precommit does not detect that I used it because of the keyword 'global'. And this keyword is necessary if I modify the value DATA_DIR in the function

Reply:
Ok, but if you only use `DATA_DIR` inside this method, you can maybe put it as a `Presenter` parameter initialized with `DATA_DIR` value, and then access to it as `self.data_dir` inside your method. What do you think?

Reply:
yes I could, and I will.
I kept it as global value so far because I put all directories in a same init file to be more organized..